login.css

```diff
@@ -155,7 +155,7 @@ p{
 
 #update{
 	color:black;
-	font-size:50px;
+	font-size:14px;
```

Comment:
04/08/2021: After trialing the font-size, I released it the font size for the forgot password link was to big. So I make an update for it to 14px which can be read clearly by people because I did a research and they suggest 14px to 16px was a reliable fonts size to use.